flycheck.el

```diff
@@ -7464,6 +7465,44 @@ Uses the GNAT compiler from GCC.  See URL
           ": " (message) line-end))
   :modes ada-mode)
 
+(flycheck-define-checker ansible-ansiblelint
+  "An Ansible linter using the ansible-lint tool.
+
+See URL `https://ansible-lint.readthedocs.io/en/latest/'."
+  ;; emacs-ansible provides ansible, not ansible-mode
+  :enabled (lambda () (seq-find (lambda (elt) (equal 'ansible elt))
```

Comment:
Couldn't we use `(bound-and-true-p ansible)` here?

Reply:
We definitely could. I didn't know about it at the time.